simple-serial-port/simple-serial-port/SimpleSerial.cpp

```diff
@@ -174,7 +130,7 @@ void SimpleSerial::CustomSyntax(const std::string &syntax_type)
     }
 }
 
-std::string SimpleSerial::ReadSerialPort(int reply_wait_time, std::string syntax_type)
+std::string SimpleSerial::ReadSerialPort(int reply_wait_time, const std::string& syntax_type)
```

Comment:
In this case i added the `const &` because SimpleSerial.cpp we had:
` std::string SimpleSerial::ReadSerialPort(int reply_wait_time, std::string syntax_type) { ....... }`
But in the definition of the class we had:
`std::string ReadSerialPort(int reply_wait_time, const std::string& syntax_type);`
So the new parameters should be matching class definition:
`std::string SimpleSerial::ReadSerialPort(int reply_wait_time, const std::string& syntax_type) { ........... }`

Reply:
Sure, I'll close this pr and change this.